.github/workflows/build.yaml

```diff
@@ -0,0 +1,34 @@
+name: 'Build & test documentation'
+
+on:
+    push:
+        branches: [ master ]
+    pull_request:
+
+jobs:
+    build:
+
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                python-version: [3.8]
```

Comment:
Why matrix?

Reply:
No particular reason.

I've used a Python template where there were multiple versions and decided that it wasn't worth it to remove it, in case we need to build these against multiple versions.